docs/src/external_mailbox_cmds.md

```diff
@@ -0,0 +1,901 @@
+
+- **Device Identification and Capabilities**
+    - Retrieve firmware versions, unique device identifiers, and device capabilities to ensure compatibility and proper configuration.
+    - Query device-specific information such as chip identifiers or subsystem details.
+
+- **Debugging and Diagnostics**
+    - Retrieve debug logs to analyze device behavior, diagnose issues, and monitor runtime states.
+    - Clear logs to reset diagnostic data and maintain storage efficiency.
+
+- **Certificate Management**
+    - Export Certificate Signing Requests (CSRs) for device keys to facilitate secure provisioning.
+    - Import signed certificates to establish a trusted certificate chain for device authentication.
+
+- **Cryptographic Services**
+    - AES encryption and decryption
+    - SHA hashing
+    - Random number generation
+    - Digital signing
+    - Signature verification
+    - Key exchange
+
+- **Debug Unlock Mechanisms**
+    - Facilitate secure debugging in production environments
+    - Ensure controlled access to debugging features
+
+## Mailbox Commands List
+
+| **Name**                          | **Command Code** | **Description**                                                                                     |
+|-----------------------------------|------------------|-----------------------------------------------------------------------------------------------------|
+| MC_FIRMWARE_VERSION               | 0x4D46_5756 ("MFWV") | Retrieves the version of the target firmware.                                                      |
+| MC_DEVICE_CAPABILITIES            | 0x4D43_4150 ("MCAP") | Retrieve the device capabilities.                                                                  |
+| MC_DEVICE_ID                      | 0x4D44_4944 ("MDID") | Retrieves the device ID.                                                                           |
+| MC_DEVICE_INFO                    | 0x4D44_494E ("MDIN") | Retrieves information about the target device.                                                     |
+| MC_EXPORT_IDEV_CSR                | 0x4D49_4352 ("MICR") | Exports the IDEVID Self-Signed Certificate Signing Request.                                        |
+| MC_IMPORT_IDEV_CERT               | 0x4D49_4943 ("MIIC") | Allows SoC to import DER-encoded IDevId certificate on every boot.                                 |
+| MC_GET_LOG                        | 0x4D47_4C47 ("MGLG") | Retrieves the internal log for the RoT.                                                            |
+| MC_CLEAR_LOG                      | 0x4D43_4C47 ("MCLG") | Clears the log in the RoT subsystem.                                                               |
+| MC_SHA_INIT                       | 0x4D43_5349 ("MCSI") | Starts the computation of a SHA hash of data.                                                      |
```

Comment:
For the cryptographic mailbox ones, I wonder if it would be easier to say a list of which ones we support (or a map of MCI -> CM) and to refer to the Caliptra runtime README for details on those commands?

I just worry about the specs getting out of sync.